### ISSUE TITLE
Changing tab on the top will reset the flashing process, resulting in a brick

### ISSUE THREAD

**Issue by ggoraa** [EdgeTX/buddy#45]
![image](https://user-images.githubusercontent.com/61655029/233743160-fa8f363b-dc97-4b43-a7f7-46e33f7ac66b.png)
If i select a tab over here while I am flashing my radio, it will reset, resulting in a bricked radio
I would suggest just disabling those tabs while flashing, with a hover tooltip explaining why it's disabled

**Comment by freshollie** [comment]
This is a good point. I need to fix the deploy process for our new hosting but happy to get this in as a feature. For the bricked radio, is it possible to unbrick?

**Comment by ggoraa** [comment]
Yea, I can easily unbrick it by just flashing again, bootloader is not affected on error

**Comment by raphaelcoeffic** [comment]
Just an idea at this point: what about folding the title bar while flashing and unfolding it once flashing is done?

**Comment by pfeerick** [comment]
I can't say I'm a fan of that idea (I don't like UIs that move stuff and then give it back). 

This is the relevant creation block for that menu:
https://github.com/EdgeTX/buddy/blob/f912b8e1caf4219091063d111cf496727bbf4b2c/src/renderer/Layout.tsx#L187-L204

I'm also relatively clueless regarding nodejs/electron, but am wondering if another style parameter can be added it that tracks a variable so that it can be disabled while at say the DeviceSelectionStep and OverviewStep steps (or perhaps just the latter?). 

i.e. if `pointer-events: none;` can be injected, it will disabled the header buttons, but leave the GitHub and settings cog active (which should be safe, as one only changes options in place, and the other opens a new browser window). Setting `pointer-events` to `auto` will restore interaction with the menu options. Just can't readily see how to add that myself :zany_face:

**Comment by freshollie** [comment]
Yes, that could be an easy option. Another way is to just add a `useBlocker` on the flashing status page to prevent navigation whilst flashing is taking place:
https://reactrouter.com/en/main/hooks/use-blocker

**Comment by raphaelcoeffic** [comment]
> Yes, that could be an easy option. Another way is to just add a `useBlocker` on the flashing status page to prevent navigation whilst flashing is taking place: https://reactrouter.com/en/main/hooks/use-blocker

Thx for the hint! I'll look into this!

**Comment by 13982720426** [comment]
How about a global popup when starting flashing and a jump warning when leaving this route? like this
![image](https://github.com/user-attachments/assets/df3d8b66-b514-4ea4-ad4c-0d8dfd5caa51)
![image](https://github.com/user-attachments/assets/0dc32d98-384e-49d2-9bb2-18d641d04935)
